src/main/java/uk/gov/register/resources/EntryResource.java

```diff
@@ -52,7 +52,8 @@ public EntryResource(RegisterReadOnly register, ViewFactory viewFactory, Request
     @Timed
     public Optional<EntryListView> findByEntryNumber(@PathParam("entry-number") int entryNumber) {
         Optional<Entry> entry = register.getEntry(entryNumber);
-        return entry.isPresent() ? Optional.of(new EntryListView(Arrays.asList(entry.get()))) : Optional.empty();
+        headerProvider.setAttachmentContentDisposition(String.valueOf(entryNumber));
+        return entry.map(entry1 -> new EntryListView(Collections.singletonList(entry1)));
```

Comment:
it is just a style preference thing, but I would rather call this `e` than `entry1`.

Reply:
Auto-generation, I didn't event read it. I'll change it.